rust/src/nfs/nfs.rs

```diff
@@ -85,6 +86,19 @@ static mut ALPROTO_NFS: AppProto = ALPROTO_UNKNOWN;
  * Transaction lookup.
  */
 
+#[derive(AppLayerFrameType)]
+pub enum NFSFrameType {
+    RPCPdu,
+    RPCHdr,
+    RPCData,
+    NFSPdu,
+    NFSStatus,
+    NFS4Pdu,
+    NFS4Tag,
+    NFS4Ops,
+    NFS4Status,
+}
```

Comment:
some `NFS4` records have `GSS` info instead of the `Tag` `Compound_Ops` Frames and wouldn't fall under this structure.
What check could be made to differentiate such cases?

e.g
```
NFS4 SETCLIENTID {
- GSS Data
- GSS Checksome
}
```